src/ConnectionConfig.test.tsx

```diff
@@ -196,7 +196,7 @@ describe('ConnectionConfig', () => {
     config.featureToggles.awsDatasourcesTempCredentials = true;
     config.awsAllowedAuthProviders = [AwsAuthType.GrafanaAssumeRole, AwsAuthType.Credentials];
     const props = getProps();
-    const overwriteOptions = { ...props.options, type: 'grafana-athena-datasource' };
+    const overwriteOptions = { ...props.options, type: 'grafana-redshift-datasource' };
```

Comment:
is this a typo?

Reply:
nope, intentional! This test demonstrates that for datasources that don't yet support temporary credentials that we don't show the new dropdown even if the feature toggle is enabled.

Previously athena was an example of a datasource that didn't have temp credentials supported, but now that's not accurate because we're adding support for temp creds in athena. So now I'm changing this to redshift since that is a datasource where temp creds is not yet supported. Eventually all our datasources will be supported and we can remove this test all together, does that make sense?

Reply:
sounds good!